App.Tests/TestWeddingService.cs

```diff
@@ -0,0 +1,180 @@
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using App.Models;
+using App.Services;
+
+namespace App.Tests
+{
+    public class TestWeddingService
+    {
+        [Fact]
+        public void testDayIndexWhenEveryoneCanCome_SundayAndSaturdayAreFree()
+        {
+            List<Guest> guests = new List<Guest>();
+            guests.Add(new Guest("Inga", new List<int>{1, 3}));
+            guests.Add(new Guest("Sasha", new List<int>{1, 2, 3, 4, 5}));
+
+            WeddingService weddingService = new WeddingService();
+            int dayIndex = weddingService.dayIndexWhenEveryoneCanCome(guests);
+            Assert.Equal(7, dayIndex);

```

Comment:
According to the requirements Saturday is preferred over Sunday so in case both days are available, it has to be Saturday

```suggestion
            Assert.Equal(6, dayIndex);
```

Reply:
blyaaa for some reasone I was sure Sunday is the most preffered. Oh no, misreading ТЗ is a bad quality for QA